src/models/propulsion/FGTurbine.cpp

```diff
@@ -592,7 +589,7 @@ void FGTurbine::bindmodel(FGPropertyManager* PropertyManager)
   PropertyManager->Tie( property_name.c_str(), this,
                         &FGTurbine::GetInjN2increment, &FGTurbine::SetInjN2increment);
   property_name = base_property_name + "/tsfc";
-  PropertyManager->Tie(property_name.c_str(), TSFC.get(), &FGParameter::GetValue);
+  PropertyManager->Tie(property_name.c_str(), &correctedTSFC);
```

Comment:
May be keep the old way so that the property is read only ?
```suggestion
  PropertyManager->Tie(property_name.c_str(), TSFC.get(), &FGParameter::GetValue);
```

Reply:
The reason I tied the property to the `correctedTSFC` member was so that there wouldn't be another function lookup when the property was requested for the case where a function was supplied for TSFC, i.e. return the cached version calculated in the last call to `run()`.

Reply:
Ah, missed the read-only distinction. Originally I meant to make it read-only, and was used to only providing a getter and not a setter, so sort of assumed I was only passing a getter reference `&correctedTSFC`, but double-checking the `Tie` class I see this form enables read and write.

Reply:
> The reason I tied the property to the `correctedTSFC` member was so that there wouldn't be another function lookup when the property was requested for the case where a function was supplied for TSFC, i.e. return the cached version calculated in the last call to `run()`.

Yes, that's a good point.
> Ah, missed the read-only distinction. Originally I meant to make it read-only, and was used to only providing a getter and not a setter, so sort of assumed I was only passing a getter reference `&correctedTSFC`, but double-checking the `Tie` class I see this form enables read and write.

Well, this can be worked around by explicitly calling `SGPropertyNode::setAttribute` on the property:
```c++
    auto node = FGPropertyManager->GetNode(property_name.c_str(), false);
    node->setAttribute(SGPropertyNode::WRITE, false);
```

Reply:
Ah, I had just reverted to providing just the `&FGParameter::GetValue` getter, convincing myself that the calls to read `tsfc` wouldn't be at the simulation rate of say 120Hz, but probably only for reporting, e.g. every 5s, or maybe 10Hz or so for graphing etc.

I'll make use of `SGPropertyNode::setAttribute` instead.